packages/libs/react-ui/README.md

```diff
@@ -127,33 +114,27 @@ export default MyApp;
 After installing dependencies, you can start Storybook with the following
 command:
 
-```
-rushx storybook
-```
+    rushx storybook
 
 ### Installation outside of the Kadena.js monorepo
 
 The component library is not yet published, to use it in an app outside of this
 mono repo you first clone this repo and then reference this library from your
 app.
 
-```
-git clone git@github.com:kadena-community/kadena.js.git
-cd kadena.js
-cd packages/libs/react-ui
-rush install
-rushx build
-cd ~/your-app-root
-```
+    git clone git@github.com:kadena-community/kadena.js.git
+    cd kadena.js
+    cd packages/libs/react-ui
+    rush install
+    rushx build
+    cd ~/your-app-root
 
 Add @kadena/react-ui as a dependency in your package.json:
 
-```
+```json
 {
-  ...
   "dependencies": {
     "@kadena/react-ui": "link:../kadena.js/packages/libs/react-ui"
-    ...
   }
 }
 ```
```

Comment:
Why code snippets sometimes with backticks sometimes with spaces? I get the algorithm for it, but not the result. Now you need to add spaces when you're changing the block without language indicator